### PR TITLE
Feature 109 funding

### PR DESCRIPTION
**Pull request by mbjones**
These are the proposed changes for Funding to use the @reverse keyword on `fundedItem`. It includes both updates to the Dataset guidance docs, a new example, a new figure, and an ADR describing the impact.

Discussion of these features was in issue #109

### CODE REVIEW

**Review by datadavev — CHANGES_REQUESTED** [review]
Overall this seems great to me as a mechanism to describe funding sources of a dataset.  There are a couple minor typos, and a suggestion to add a reference to the spec for `@reverse`. Otherwise this is good to go.

The only issues I have are in the broader context of assuming that there's some special characteristic of the JSON structure of the JSON-LD block that confers meaning, when in fact, there is none. The logical representation of the content using `@reverse` or `@graph` with appropriate targets is the same, and the RDF representation of each is identical. Hence, my impression is that emphasis should be more on the correct representation of the relations between entities rather than promoting one particular construct (i.e. `@reverse`) when there is no logical difference.

**Comment by mbjones** [comment]
Yeah, makes sense. How about if I add a sentence at the end explaining that and alternative would be to use the `@graph` representation?

**Comment by datadavev** [comment]
> Yeah, makes sense. How about if I add a sentence at the end explaining that and alternative would be to use the `@graph` representation?

Kind of feels more like there should be a general guideline on JSON-LD, but adding a sentence here could be helpful. Perhaps something like:

> Note that using `@reverse` is a convenience that can simplify the construction of JSON-LD for describing datasets since it places the funders as objects within the `SO:Dataset` object rather than composing multiple objects within a `@graph` list which may be more complicated to generate. Logically there is no difference. In either case, consumers of JSON-LD should take appropriate steps to process the content to achieve an expected JSON structure (e.g. with [framing](https://www.w3.org/TR/json-ld11-framing/)) or treat the JSON-LD as an RDF graph and query accordingly.

File: guides/Dataset.md [hunk 1]
```diff
@@ -960,98 +960,63 @@ Back to [top](#top)
...
-  ...
-}
-</pre>
+Data providers should include funding information in their Dataset descriptions to enable discovery and cross-linking. The information that would be useful includes the title, identifier, and url of the grant or award, along with structured information about the funding organization, including its name and identifier. Organizational identifiers are best represented using either a general purpose institutional identifier such as a [ROR](https://ror.org), [GRID](https://grid.ac/), or ISNI identifier, or a more detailed [Crossref Funder ID](https://api.crossref.org/funders/). The ROR for the National Science Foundation (https://ror.org/021nxhr62), for example, provides linkages to related identifiers as well. The Funder ID has the advantage that it inlcudes both agency funders like the National Science Foundation (http://dx.doi.org/10.13039/100000001), but also provides identifiers for individual funding programs within those agencies, such as the NSF GEO Directorate (https://api.crossref.org/funders/100000085). When possible, providing both a ROR and FunderId is helpful. Here's an example of identifiers for the National Science Foundation:
```
[the remark this answers is not in the document]
Comment (mbjones) [reply guides/Dataset.md]: Fixed in sha #82a734b.

**Review by smrgeoinfo — COMMENTED** [review]
In the first paragraph on Funding,  the Crossref Funder ID is (I think...) referred to with three different names. 

> ...institutional identifier such as a ROR, or ISNI identifier, or a more detailed **Crossref Funder** ID. The ROR for the National Scien....ifiers as well. The **Funder ID** has the advan.... When possible, providing both a ROR and **FunderId** is helpful. ...

It would be much clearer if the same name was used, maybe 'Funder ID' and link to Crossref (https://www.crossref.org/services/funder-registry/) the first time its used?

**Comment by mbjones** [comment]
Thanks, @smrgeoinfo, good catch. I just harmonized the language used to "Funder ID" and added a link in sha e87b2e0. Does that address your feedback?

**Comment by smrgeoinfo** [comment]
looks good

**Pull request merged**